array-methods/1-array.js

```diff
@@ -45,6 +119,26 @@ var customers = [
 ];
 // Find all customers whose firstname starts with 'J'
 
+var name_with_j = customers.filter(name => {
+  name.firstname[0] == "J";
+})
+
+var name_with_j = customers.filter(name => {
+  return name.firstname.includes("J",0);
+})
+console.log(name_with_j);
+
```

Comment:
Don't use `_` in javascript naming conventions. Refactor your code to use `camelCasing`